Mostly minor suggestions (#3)
cambiatus/grant-framework: applications/cambiatus.md

- [Wireframe in production](https://drive.google.com/file/d/1yr7H5RMlMqKqgawuBTvsnA9f_1otgQ90/view?usp=share_link)

> This grants work is a follow up on a previous grant. For that reason, it's important to outline our data models with some documentation that can be found [here](https://github.com/cambiatus/contracts/blob/master/community/community.hpp)

Reviewer: A ideia que queria transmitir aqui é a seguinte, temos um Grant anterior, funcionou, avançamos, ele deu resultados e os resultados são esses: (um monte de código e links pra repositórios)

Reviewer: não sei se esse trecho passa essa ideia bem, relendo, oq acham @MatheusBuss @henriquecbuss

Reviewer 2: Talvez algo como

```suggestion
This grant builds on top of another successful [grant](https://pomelo.io/grants/cambiatus) we worked on. Because of that, we already have some data models and documentation, which can be found [here](https://github.com/cambiatus/contracts/blob/master/community/community.hpp) and in the following code snippets:
```

Reviewer: show, adorei bro, valeu!

  ACTION upsertrole(eosio::symbol community_id, eosio::name name, std::string color, std::vector<std::string> & permissions);

  /// @abi action

>   /// Sets a number of roles for a user

Reviewer: opa, esse erro tá no contrato então kkkkk

Author: É bem contra intuitivo mesmo, tive que pesquisar

- [Our architecture](https://www.figma.com/file/i4Azmx3APcQulJ1zmHNLIV/Cambiatus-Architecture?node-id=0%3A1), which can help other projects decide on their own path

### PoC/MVP or other relevant prior work or research on the topic

> Our current structure includes the following roles: members, validators and admins. These roles are beign used by our over 9000 users and have been used by them since 2018: 

Reviewer: ```suggestion
Our current structure includes the following roles: members, validators and admins. These roles are beign used by our over 9000 users since 2018: 
```

Reviewer 2: Enquanto estamos aqui, podemos arrumar o `beign` -> `being`

Author: Nice catch!

    - People from our team are long time learners and teachers of the ecosystem (our team facilitated 3 EOS hackathons and many meetups in Brazil since 2018).
    - In 2022 we got the first financial support from the EOS Community via Pomelo, when we reached the top 10 projects with the roles and permissions feature, base work for what we are talking about on this grant. 

<br/>

- Who is your target audience?
    - The whole EOSIO/ Antelope compatible ecosystem (Wax, Ultra, Telos, etc),
    - Our 8 communities in production, its leadership teams and their 9k+ users from Costa Rica, Brazil and Ethiopia,
    - Dapp developers trying to add a comprehensive permission tool to their smart contracts,
    - DAOs facing the limitations of a simple holder whale taking important decisions,
    - Users with little to no blockchain experience

<br/>

- What need(s) does your project meet?
  - **Decentralization:** using Roles & Permissions, community leaders will be able to assign roles and permissions to other people involved in community leadership and by doing so, taking the decision/task away from just one person.

>   - **Decentralized governance:** allowing decision-making to be inclusive and collaborative, facilitating a scenario in which people can exercise their voting power based on their history of living within the community, instead of having their voting power stipulated only by the number of tokens you own. New leaderships can emerge as time progresses. 

Reviewer: nice!

### Team Experience

> Our team has been building on EOSIO for 4 years now. We've done our pilot in a [community in Costa Rica](https://www.cambiatus.com/verdes), which is still ongoing. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including [smart contract](https://github.com/cambiatus/contracts), [backend](https://github.com/cambiatus/backend), [infrastructure](https://github.com/cambiatus/backend#nginx-setup), wallet integrations, [frontend](https://github.com/cambiatus/frontend) and finally our own wallet inside our app.

Reviewer: ```suggestion
Our team has been building on EOSIO for 4 years now. We've done our pilot in a community in Costa Rica. We've been part of every major announcement since before Block.one released version 1.0. Since then we've deployed and maintained our own private network, developed several features into our app, including [smart contract](https://github.com/cambiatus/contracts), [backend](https://github.com/cambiatus/backend), [infrastructure](https://github.com/cambiatus/backend#nginx-setup), wallet integrations, [frontend](https://github.com/cambiatus/frontend) and finally our own wallet inside our app.
```

Reviewer: our pilot community in Costa Rica was the Pulperos app, done in partnership with Fundes, a local ong. Its not ongoing anymore

Author: Oops, honest mistake, sorry

> Although we have similarities with projects like Hypha, Cambiatus is different because our main focus is on social currencies and financial resiliency to foster planetary regeneration. This is shown through our actions, such as choosing to develop a mobile-first webApp because many of our members don’t have enough memory space in their smartphones to download an app, others don't have access to a PC/laptop and others yet have a slow and unreliable internet connection. These actions are reflections of our goals and learnings with our partner communities which are located in Latin America and Africa.

Reviewer: ```suggestion
Although we admire and like projects like Hypha, Cambiatus is different because our main focus is on social currencies and financial resiliency to foster planetary regeneration. This is shown through our actions, such as choosing to develop a mobile-first webApp because many of our members don’t have enough memory space in their smartphones to download an app, others don't have access to a PC/laptop and others yet have a slow and unreliable internet connection. These actions are reflections of our goals and learnings with our partner communities which are located in Latin America and Africa.
```

Reviewer: a ideia era transmitir que gostamos e apreciamos o Hypha

Author: Show então!

> We also have translations to portuguese, spanish, catalan, amharic and english. Our educational content is in english, translated to portuguese and spanish - and we have community support in those three languages. Being pluricultural and language inclusive to our communities is a must.

Reviewer: nice! kkkk

> Our biggest differential to our communities is that all they're all provided with assisted tokenomics co-design sessions - we don’t assume they know how to create a currency or develop their own tokenomics. We added a human layer to enable our partner communities to use our tools and be able to promote change in their economic context. It's a constructivist method, allowing co-creation, co-organization and aligning objectives under a group.

Reviewer 2: ```suggestion
Our biggest differential to our communities is that they're all provided with assisted tokenomics co-design sessions - we don’t assume they know how to create a currency or develop their own tokenomics. We added a human layer to enable our partner communities to use our tools and be able to promote change in their economic context. It's a constructivist method, allowing co-creation, co-organization and aligning objectives under a group.
```
